packages/web/src/App.css

```diff
@@ -75,6 +75,10 @@ html, body, #root {
   padding-left: 10px;
 }
 
+.add-player-form {
```

Comment:
Unused class selector

Reply:
Thank you for all the good feedback.

I had initially set myself up to style the component and then decided to wait.